astrocut/cube_cut.py

```diff
@@ -668,15 +672,12 @@ def _apply_header_inherit(self, hdu_list):
                         hdu.header[kwd] = (primary_header[kwd], primary_header.comments[kwd])
             
 
-    def _build_tpf(self, product, cube_fits, img_cube, uncert_cube, cutout_wcs_dict, aperture):
+    def _build_tpf(self, cube_fits, img_cube, cutout_wcs_dict, aperture, uncert_cube=None):
         """
         Building the cutout target pixel file (TPF) and formatting it to match TESS pipeline TPFs.
 
         Paramters
         ---------
-        product : str
-            The product type to make the cutouts from.
-            Can either be 'SPOC' or 'TICA'.
         cube_fits : `~astropy.io.fits.hdu.hdulist.HDUList`
             The cube hdu list.
         img_cube : `numpy.array`
```

Comment:
(this comment is for line number 685, GH doesn't allow comments on collapsed sections of code)

`uncert_cube` docstring needs to be updated

in general, we should update the docstrings everywhere we have changed the signature of the functions